### DISCUSSION TITLE
Laravel Google Chat Log

### DISCUSSION THREAD

**Discussion by theriddleofenigma** [HassanZahirnia/laravel-package-ocean#129] [Package Suggestions]
Brings up the option for sending the logs to google chat [Google Workspace formerly called GSuite] from [Laravel](https://laravel.com/)/[Lumen](https://lumen.laravel.com/).

https://github.com/theriddleofenigma/laravel-google-chat-log

Have 15k+ Installs,
https://packagist.org/packages/theriddleofenigma/laravel-google-chat-log/stats

The error log will look like,
<img width="1353" alt="reference" src="https://github.com/HassanZahirnia/laravel-package-ocean/assets/29883026/59ef5a3a-c43a-4946-be0d-73bc3210d052">

**Comment by HassanZahirnia** [comment]
Hey! Thanks for your package suggestion!

Does the Google chat/workspace charge the user for this feature?

Also if you want this package added, you need to make it compatible with the latest Laravel version (v10).

**Reply by theriddleofenigma** [reply]
No, it's free of cost. If anyone have a workspace account then they can add a webhook in any space-group and map it in env.
Basically, Google workspace provides webhooks in chat with any subscription. This package utilize the webhook.

For v10 support, probably will make a release by this weekend. Since it has some breaking changes with monolog version, it will become v2.0.

**Reply by theriddleofenigma** [reply]
@HassanZahirnia, A new version has been released v2.0 and now it supports the latest Laravel version v10. We would be very much happy if you added the package to the website. Thank you!

**Reply by HassanZahirnia** [reply]
I've added the package! Thanks for the update!

**Reply by kumaravelabinbev** [reply]
Thank you 😃
